### ISSUE TITLE
Consider PXIDEV for NCCH header dumping

### ISSUE THREAD

**Issue by neobrain** [neobrain/braindump#11]
If we have access to [PXIDEV:ReadCTRCARD_Cmd40](http://www.3dbrew.org/wiki/PXIDEV:ReadCTRCARD_Cmd40), we might be able to dump the raw cartridge even from userspace (assuming that the unknown 0x10 bytes are a raw cart command like Normmatt suggested). The largest part of the dump would still be encrypted, but the (unencrypted) NCCH header could be extracted.

**Comment by mariomadproductions** [comment]
Could this allow dumping of encrypted NCCH header? (And therefore 1:1 cart dumps)?

**Comment by neobrain** [comment]
Yes, but chances are this command is not accessible from any userland application.

**Comment by dazjo** [comment]
Most games have access to pxi:dev.
